### ISSUE TITLE
Some questions about Inference on onsets and frames model

### ISSUE THREAD

**Issue by wwpww** [tensorflow/magenta#1603]
First, I use MAPS dataset as inference material, just ENSTDkAm/MUS this part.
The test.tfrecord-00003-of-00004 just seems to loss some data when inference.
And using [this code](https://github.com/tensorflow/magenta/tree/master/magenta/models/onsets_frames_transcription#maps-dataset-optional) converts dataset to TFrecord file.
Then by pre-trained [checkpoint](https://storage.googleapis.com/magentadata/models/onsets_frames_transcription/maestro_checkpoint.zip) infering adding the argument `--preprocess_examples=True`.
It works, but `tensorboard --logdir="${OUTPUT_DIR}"`, I got some results, 
![metrics1](https://user-images.githubusercontent.com/48265203/66396432-53ec1a00-ea0c-11e9-90e1-c24f2db748b2.png)
So, where am I wrong? Thanks!

**Comment by cghawthorne** [comment]
What happens if you do inference over all the shards of the test.tfrecord file? Do they all get scores that low?

**Comment by wwpww** [comment]
> What happens if you do inference over all the shards of the test.tfrecord file? Do they all get scores that low?

I don't know, I just take a part of dataset, must I use all the dataset?

**Comment by cghawthorne** [comment]
You certainly aren't required to use all of the dataset, I'm just curious if the poor results are specific to that shard or if they're consistent across the entire dataset.

**Comment by wwpww** [comment]
OK, because of my hardware facility, maybe I have to give up this. 
Thanks!
I really want to know whether the GraphDef file is the same as the full train when I train onsets&frames model just a while. I want to use GraphDef file and checkpoint to generate frozen GraphDef file, and finally, get TFlite file.
Maybe my words is not clear.
Thank you for your hospitality！

**Comment by cghawthorne** [comment]
The contents of the GraphDef file should not be affected by how long the model has trained. However, there are a number of other barriers to getting things to work with TFLite, though it's definitely possible. We're also hoping to look at a realtime/TFLite version soon, but not sure when it would be available.

**Comment by nkjulia** [comment]
> First, I use MAPS dataset as inference material, just ENSTDkAm/MUS this part.
> The test.tfrecord-00003-of-00004 just seems to loss some data when inference.
> And using [this code](https://github.com/tensorflow/magenta/tree/master/magenta/models/onsets_frames_transcription#maps-dataset-optional) converts dataset to TFrecord file.
> Then by pre-trained [checkpoint](https://storage.googleapis.com/magentadata/models/onsets_frames_transcription/maestro_checkpoint.zip) infering adding the argument `--preprocess_examples=True`.
> It works, but `tensorboard --logdir="${OUTPUT_DIR}"`, I got some results,
> ![metrics1](https://user-images.githubusercontent.com/48265203/66396432-53ec1a00-ea0c-11e9-90e1-c24f2db748b2.png)
> So, where am I wrong? Thanks!
hey, i  got such results too,  how u solve it？ 

INFO:tensorflow:Saving dict for global step 100: global_step = 100, loss = 0.39757368, losses/frame = 0.21719432, losses/offset = 0.048358064, losses/onset = 0.04870561, losses/velocity = 0.07781867, metrics/frame_accuracy = 0.9231014, metrics/frame_accuracy_without_true_negatives = 0.0, metrics/frame_f1_score = 0.0, metrics/frame_precision = 0.0, metrics/frame_recall = 0.0, metrics/note_density = 0.0, metrics/note_f1_score = 0.0, metrics/note_precision = 0.0, metrics/note_recall = 0.0, metrics/note_with_offsets_f1_score = 0.0, metrics/note_with_offsets_precision = 0.0, metrics/note_with_offsets_recall = 0.0, metrics/note_with_offsets_velocity_f1_score = 0.0, metrics/note_with_offsets_velocity_precision = 0.0, metrics/note_with_offsets_velocity_recall = 0.0, metrics/note_with_velocity_f1_score = 0.0, metrics/note_with_velocity_precision = 0.0, metrics/note_with_velocity_recall = 0.0
I0525 11:41:23.122856 139871819388672 estimator.py:2066] Saving dict for global step 100: global_step = 100, loss = 0.39757368, losses/frame = 0.21719432, losses/offset = 0.048358064, losses/onset = 0.04870561, losses/velocity = 0.07781867, metrics/frame_accuracy = 0.9231014, metrics/frame_accuracy_without_true_negatives = 0.0, metrics/frame_f1_score = 0.0, metrics/frame_precision = 0.0, metrics/frame_recall = 0.0, metrics/note_density = 0.0, metrics/note_f1_score = 0.0, metrics/note_precision = 0.0, metrics/note_recall = 0.0, metrics/note_with_offsets_f1_score = 0.0, metrics/note_with_offsets_precision = 0.0, metrics/note_with_offsets_recall = 0.0, metrics/note_with_offsets_velocity_f1_score = 0.0, metrics/note_with_offsets_velocity_precision = 0.0, metrics/note_with_offsets_velocity_recall = 0.0, metrics/note_with_velocity_f1_score = 0.0, metrics/note_with_velocity_precision = 0.0, metrics/note_with_velocity_recall = 0.0
INFO:tensorflow:Saving 'checkpoint_path' summary for global step 100: /DATA/disk1/juliayang/data/maestro-v3/test_maestro/onsets_frames/model.ckpt-100
